src/inc/corjit.h

```diff
@@ -123,16 +123,14 @@ enum CorJitFlag
 
 #ifdef MDIL
     CORJIT_FLG_MDIL                = 0x00004000, // Generate MDIL code instead of machine code
-#else // MDIL
-    CORJIT_FLG_CFI_UNWIND          = 0x00004000, // Emit CFI unwind info
-#endif // MDIL
 
-#ifdef MDIL
     // Safe to overlap with CORJIT_FLG_MAKEFINALCODE below. Not used by the JIT, used internally by NGen only.
     CORJIT_FLG_MINIMAL_MDIL        = 0x00008000, // Generate MDIL code suitable for use to bind other assemblies.
 
     // Safe to overlap with CORJIT_FLG_READYTORUN below. Not used by the JIT, used internally by NGen only.
     CORJIT_FLG_NO_MDIL             = 0x00010000, // Generate an MDIL section but no code or CTL. Not used by the JIT, used internally by NGen only.
+#else // MDIL
+    CORJIT_FLG_CFI_UNWIND          = 0x00004000, // Emit CFI unwind info
 #endif // MDIL
```

Comment:
> Why did you change this?
This was part of the earlier change (which had `FLG_USE_PINVOKE_HELPERS` instead of `FLG_CFI_UNWIND`); I kept it when I rebased.

>  It was intentional as it was, to keep identical bit patterns together.

If we believe that keeping identical bit patterns together is important, then we should apply that to the rest of the file (e.g. the definitions for `CORJIT_FLG_MAKEFINALCODE` and `CORJIT_FLG_READYTORUN` should be next to `CORJIT_FLG_MINIMAL_MDIL` and `CORJIT_FLG_NO_MDIL`, respectively). I'm fine with either keeping all of the bits that are affected by an particular symbol together or keeping the bits themselves together. I have a minor preference for the former since it makes it easier to understand what changes when a given symbol is defined. Note that we already use that style for processor-specific preprocessor symbols.